src/custom/pages/Claim/index.tsx

```diff
@@ -239,8 +251,24 @@ export default function Claim() {
     // setActiveClaimAccount and other dispatch fns are only here for TS. They are safe references.
   }, [account, isSearchUsed, setActiveClaimAccount, setInvestFlowStep, setIsInvestFlowActive])
 
+  // Transaction confirmation modal
+  const { TransactionConfirmationModal, openModal, closeModal } = useTransactionConfirmationModal(
+    OperationType.APPROVE_TOKEN
+  )
+
+  const [approveState, approveCallback] = useApproveCallbackFromClaim(
```

Comment:
I don't fully get it. Why do you need a special approval callback for claims?
Isn't it like regular approvals (approve max always)?

Reply:
guess it doesn't really, @anxolin made a comment about it. I'll look into reverting if it's a big issue